frame/elections-phragmen/src/lib.rs

```diff
@@ -334,27 +399,62 @@ decl_module! {
 		///
 		/// A defunct voter is defined to be:
 		///   - a voter whose current submitted votes are all invalid. i.e. all of them are no
-		///     longer a candidate nor an active member.
+		///     longer a candidate nor an active member or a runner-up.
+		///
+		///
+		/// The origin must provide the number of current candidates and votes of the reported target
+		/// for the purpose of accurate weight calculation.
 		///
 		/// # <weight>
-		/// #### State
-		/// Reads: O(NLogM) given M current candidates and N votes for `target`.
-		/// Writes: O(1)
+		/// No Base weight based on min square analysis.
+		/// Complexity of candidate_count: 1.755 µs
+		/// Complexity of vote_count: 18.51 µs
+		/// State reads:
+		///  	- Voting(reporter)
+		///  	- Candidate.len()
+		///  	- Voting(Target)
+		///  	- Candidates, Members, RunnersUp (is_defunct_voter)
```

Comment:
this does not only reads 3 values but at most 3*defunct.vote_count

Reply:
can you explain a bit more: under the hood, we are searching inside these 3 storage items, all of which are `StorageValue` and contain a vector, so from the standpoint of just _count_ of storage reads, I think it is 3.

Reply:
ah you are right, I missread.